internal/storage/fs/sync.go

```diff
@@ -22,69 +22,109 @@ func (s *syncedStore) GetFlag(ctx context.Context, namespaceKey string, key stri
 	s.mu.RLock()
 	defer s.mu.RUnlock()
 
+	if namespaceKey == "" {
+		namespaceKey = flipt.DefaultNamespace
+	}
```

Comment:
would you mind adding tests for `default` namespace for fs stores?

Reply:
@markphelps Yeah let me do that. Apologies!